### ISSUE TITLE
fonts.googleapis.com is not loading / taking so long to load

### ISSUE THREAD

**Issue by rjshrjndrn** [arkenfox/user.js#1083]
![image](https://user-images.githubusercontent.com/2563385/103258502-21ba3580-498d-11eb-9296-9372b9a7e253.png)

for example:
visit this url
https://carniceriadomicilio.cl/docs/e2p4r.php?id=819012-opus-vs-flac
or 
docs.ansible.com
![image](https://user-images.githubusercontent.com/2563385/103258649-e1a78280-498d-11eb-966d-ad8b33152494.png)
After sometime, when it loads, the fonts url is getting loaded via http
![image](https://user-images.githubusercontent.com/2563385/103258686-1ca9b600-498e-11eb-95af-269387153307.png)

**Comment by Thorin-Oakenpants** [comment]
https://docs.ansible.com/ - practically a vanilla (in Nightly), no extensions loaded, and I do not even see google fonts
![ansible](https://user-images.githubusercontent.com/16656956/103280390-3536c200-49c8-11eb-8786-97e2d190595e.png)

**Comment by earthlng** [comment]
works for me in a fresh profile with the latest master user.js:

![1](https://user-images.githubusercontent.com/25846669/103290395-55bf4600-49e1-11eb-83b1-b755c735fd90.png)

must be one of your overrides or extensions

**Comment by Thorin-Oakenpants** [comment]
How come I don't even attempt to get fonts.goolag? ETP is on with dFPI by default, but I don't see goolag listed in the blocked items

**Comment by rjshrjndrn** [comment]
I think it's intermittent.. Some goofup with http/https
The above test is vanila profile, vanila user.js enabled. No overrides or plugins.
I'm gettng https://4anime.to also **ssl issue**. So I think it's something related to ssl. Not sure though..

![image](https://user-images.githubusercontent.com/2563385/103329159-6dcfad80-4a53-11eb-8024-49b049e3394b.png)

in Brave it's working

![image](https://user-images.githubusercontent.com/2563385/103329173-863fc800-4a53-11eb-8e38-6f3c97d255d1.png)

**Comment by rjshrjndrn** [comment]
If somebody faces this issue.
This is how I resolved it
```
user_pref("network.dns.disableIPv6", false); //1701
```
I don't know the privacy/security implications. But this is a trade of I'm willing to make

**Comment by Thorin-Oakenpants** [comment]
thats whacky. I wonder what happens if you use the default user.js but disabled HTTPS-only mode (item 1244) dom.security.https_only_mode

**Comment by rjshrjndrn** [comment]
Created a new profile.
Downloaded the latest master user.js
made 1244 as false. and I was still getting the same error.
And for software proxy, no. I'm not using any. 
But for the internet, I'm using a Jio dongle (India).

**Comment by earthlng** [comment]
thanks for reporting the problem and a possible solution.

Can you please do 1 more test?

- start Firefox with a new profile and an unchanged master user.js
- load the anime page and note that it doesn't work
- load about:config in another tab
- **without** restarting FF, reset the IPv6 pref in about:config and then reload the anime tab. Does it work now?
- again **without** restarting FF, toggle the pref again and reload the anime tab. Is it broken again?
- **without** restarting FF, reset the pref again and then reload the anime tab. Does it now work again?

in short, can you reliably and repeatedly turn the problem on and off by just toggling the pref on/off and without restarting Firefox?

If that's the case, I can add the pref to our troubleshooter script. But if there's some kind of cache involved and the problem doesn't reappear after it worked for the 1st time, I don't want to add the pref to the script because it could lead to false or misleading results. Thanks

**Comment by Thorin-Oakenpants** [comment]
This is the bit that gets me "After sometime, when it loads, the fonts url is getting loaded via http" - the user.js is using HTTPS-only mode.

**Comment by rjshrjndrn** [comment]
> thanks for reporting the problem and a possible solution.
> 
> Can you please do 1 more test?
> 
>     * start Firefox with a new profile and an unchanged master user.js
> 
>     * load the anime page and note that it doesn't work
> 
>     * load about:config in another tab
> 
>     * **without** restarting FF, reset the IPv6 pref in about:config and then reload the anime tab. Does it work now?

No

(I've commented 1701 in user.js, so that the change I made shouldn't override)

**with** restart FF, it worked
 
>     * again **without** restarting FF, toggle the pref again and reload the anime tab. Is it broken again?

No

**with restart** FF, the site is broken

>     * **without** restarting FF, reset the pref again and then reload the anime tab. Does it now work again?

No

**with restart**, it worked

I guess we need a restart for this change to reflect ?

**Comment by earthlng** [comment]
ok, thanks for testing it again. If it requires a restart it's not a suitable pref for the troubleshooter script. Good to know, thanks!

**Comment by rjshrjndrn** [comment]
I guess, can close the issue @Thorin-Oakenpants ??

**Comment by Thorin-Oakenpants** [comment]
I'm still not convinced. Why does the HTTPS-Only mode eventually get the fonts via HTTP. I think the IPv6 is a red herring and fixes a symptom, not the actual problem

**Comment by rjshrjndrn** [comment]
@Thorin-Oakenpants do you want me to do any more testing ? I'll be happy to help.

**Comment by Thorin-Oakenpants** [comment]
Not sure if you can. What I suspect is the problem is the ISP (I don't know enough about Jio dongle, sorry) and/or HTTPS-Only mode (HOM). If you can definitively prove that HoM is breaking, then we could upstream it at Mozilla

**Comment by Thorin-Oakenpants** [comment]
@rjshrjndrn  I just want to clear something up - you said you got an error (PR_CONNECT_RESET_ERROR) with https://4anime.to - did the IPv6 pref change fix that?

**Comment by rjshrjndrn** [comment]
yes it did @Thorin-Oakenpants . I'm no means expert. But that was I observed.
https://github.com/arkenfox/user.js/issues/1083#issuecomment-752903271

**Comment by Thorin-Oakenpants** [comment]
^^ thanks ... that just makes me more inclined to think it's an Server<->ISP related issue

**Comment by rjshrjndrn** [comment]
Seems like we've no definitive root cause but have one weird issue in hand, and a possible solution. 
So shall we close the thread then? @Thorin-Oakenpants

**Comment by Thorin-Oakenpants** [comment]
you're fine: you have something that fixes it at your end, but I want to leave this open to remind me to follow up on it. Thanks for troubleshooting 👍
